tools/ci_build/github/azure-pipelines/azure-pipelines-py-packaging.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       displayName: 'Run build script'
       inputs:
         scriptPath: 'tools/ci_build/github/linux/run_dockerbuild.sh'
-        args: '-c Release -o ubuntu16.04 -d gpu -r $(Build.BinariesDirectory) -p $(python.version) -x "--build_wheel --use_mkldnn"'
+        args: '-c Release -o ubuntu16.04 -d gpu -c cuda9.1-cudnn7.1 -r $(Build.BinariesDirectory) -p $(python.version) -x "--build_wheel --use_mkldnn"'
```

Comment:
did you mean to say cuda10 here?

Reply:
for the python package I think we are still releasing with cuda 9.1 for Azure?

---
In reply to: [239301044](https://github.com/Microsoft/onnxruntime/pull/112#discussion_r239301044) [](ancestors = 239301044)

Reply:
yeah, not sure if Azure upgraded to cuda 10. @vinitra might know.